admin/cleanup/FixTrackLength.pl

```diff
@@ -58,10 +59,10 @@
        JOIN track t ON t.medium = m.id
       WHERE t.length IS NULL OR t.length = 0 AND m.track_count > 0
         AND (mf.has_discids = TRUE OR mf.has_discids IS NULL)'
-) };
-printf localtime() . " : Found %d medium%s\n",
```

Comment:
There seem to be some syntax errors introduced in this script, but I didn't full investigate: try running `./admin/cleanup/FixTrackLength.pl --dry-run`.

Reply:
Good catch, fixed all of those in commit https://github.com/metabrainz/musicbrainz-server/pull/2272/commits/cf59e5b53fd15776698710ebe9902ac348e5ba2b.